finalize-test.jl

```
@@ -25,7 +27,7 @@ function start()
 
     for jobid in keys(jobdict)
         filename = joinpath(slurmoutdir, jobid)
-        files["out_$jobid"] = Dict("content" => String(read(filename)))
+        files["out_$jobid"] = Dict("content" => String(read(filename, MAX_LOG_SIZE)))
```

would we want to upload the tail of the log (seek back from end MAX_LOG_SIZE)?

---

maybe? though usually the failure occurs earlier in the log, and the rest is noise.

---

maybe I don't quite understand why a job would error and then not fail immediately

---

> can we turn off pkg verbose=true?

It is occasionally useful, as otherwise build failures are hidden.

---

i guess if there isn't a better way to do this...

---

https://github.com/JuliaLang/Pkg.jl/issues/1870

---

I think it's caused by instantiate, not build.

---

it's actually curl. I've just modified my `~/.curlrc` file for now to add `--silent`